codice/ddf
# DDF-2177 Add Auto Push configuration property to the registry store

#### What does this PR do?
Add an Auto-Publish(Auto Push) Configuration property to the RegistryStore. This configuration is accessible in the add/edit modal on the remote registries tab in the registry app ui.

There are also fixes to subscription and publish services within the registry that manifested when this functionality was added.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@gordocanchola @vinamartin @ani6gup @clockard 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl @figliold 

#### How should this be tested?
Install registry app (with security) and connect to a second registry by adding it in the remote registries tab of the registry app with Auto-Push on. Observe presence of registry identity metacard when querying metacards in the second DDF's solr once the remote becomes 'available' on the first registry. 

#### Any background context you want to provide?
Minor UI updates are in this PR fixing the remote name as well as adding the autoPush to the registry model/modal. 

#### What are the relevant tickets?
DDF-2177 Add a configuration property to the registry store, 'Auto push identity node', which is disabled by default	

If the auto publish is enabled and push is enabled then the registry store will publish the local identity node to the remote registry when the it becomes available.

#### Screenshots (if appropriate)
![screen shot 2016-06-23 at 3 29 34 pm](https://cloud.githubusercontent.com/assets/11355332/16322345/5730eb1e-3957-11e6-9869-a96af2201aba.png)

#### Checklist:
